src/components/Coverage/Coverage.js

```diff
@@ -78,11 +79,17 @@ export default class Coverage extends React.Component {
 
     if (isMonograph) {
       return (
-        <MonographCoverage pci={dataToRender} />
+        <div>
+          <MonographCoverage pci={dataToRender} />
+          <Embargo embargo={dataToRender.embargo} />
+        </div>
       );
     } else {
       return (
-        <SerialCoverage statements={dataToRender.coverage} />
+        <div>
+          <SerialCoverage statements={dataToRender.coverage} />
+          <Embargo embargo={dataToRender?.embargo ?? dataToRender?._object?.embargo ?? dataToRender?.resource?._object.embargo} />
```

Comment:
Logic here is way too confusing. Can you just do this in the `if` blocks above? Like create a `let embargo` variable and populate them accordingly in the `if` blocks based on wether its a line, eresource or a pci